lib/components/dropdown-header.js

```diff
@@ -0,0 +1,23 @@
+import { mergeData } from "../utils";
+
+export const props = {
+    tag: {
+        type: String,
+        default: "h6"
+    }
+};
+
+export default {
+    functional: true,
+    props,
+    render(h, { props, data, children }) {
+        return h(
+            props.tag,
+            mergeData(data, {
+                staticClass: "dropdown-header",
+                attrs: { tabindex: "-1" }
```

Comment:
The main dropdown mixin doesn't focus headers any more.  users will need to point aria-describedby attribute to a  header with an ID.